proto/rill/runtime/v1/resources.proto

```diff
@@ -121,6 +121,11 @@ message ModelSpec {
   google.protobuf.Struct stage_properties = 17;
   string output_connector = 1;
   google.protobuf.Struct output_properties = 12;
+  // retry is optional.
+  optional uint32 retry_attempts = 26;
```

Comment:
The comment doesn't really add anything here given there's a keyword saying `optional`